src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F

```diff
@@ -215,6 +224,24 @@ subroutine cloudiness_from_MPAS(itimestep,configs,mesh,diag_physics,sfc_input,it
        enddo
 
     case default
+
+    if (convection_scheme=="cu_grell_freitas") then
+       call mpas_pool_get_array(diag_physics,'cldfrac_cu',cldfrac_cu)
+       call mpas_pool_get_array(diag_physics,'qi_cu',qi_cu)
+       call mpas_pool_get_array(diag_physics,'qc_cu',qc_cu)
+       do j = jts,jte
+       do k = kts,kte
+       do i = its,ite
+          if (cldfrac_cu(k,i) .gt. 0.001) then
+             cldfrac_p(i,k,j) = cldfrac_p(i,k,j) + cldfrac_cu(k,i)
```

Comment:
It looks like the cloud fraction can become > 1. Maybe take the max? or limit it to be <= 1.

Reply:
@joeolson42 Thanks for your suggestion. Yes, I will limit it <=1 and update the PR accordingly.